contracts/margin/external/BucketLender/EthWrapperForBucketLender.sol

```diff
@@ -93,4 +102,46 @@ contract EthWrapperForBucketLender
         // deposit the tokens
         return BucketLender(bucketLender).deposit(beneficiary, amount);
     }
+
+    /**
+     * Allows users to send eth directly to this contract and have it be wrapped and sent to a
+     * BucketLender to be lent for some margin position.
+     *
+     * @param  bucketLender  The address of the BucketLender contract to deposit money into
+     * @return               The bucket number that was deposited into
+     */
+    function withdrawEth(
+        address bucketLender,
+        uint256[] buckets,
+        uint256[] maxWeights
+    )
+        external
```

Comment:
Do these functions need to be `nonReentrant`? Personally, I think they don't. Just wanted to surface for visibility